packages/babel-helper-function-name/src/index.js

```diff
@@ -45,6 +45,18 @@ const visitor = {
   },
 };
 
+function getNameFromLiteralId(id) {
+  if (t.isNullLiteral(id)) {
+    return null;
+  } else if (t.isRegExpLiteral(id)) {
+    return `_${id.pattern}_${id.flags}`;
+  } else if (t.isTemplateLiteral(id) && id.expressions.length === 0) {
```

Comment:
It seems like it would be best to concat the quasis all together for this value.